### DISCUSSION TITLE
LU factorization with complete pivoting

### DISCUSSION THREAD

**Discussion by wdecre** [giaf/blasfeo#144] [Ideas]
In one of our projects we want to perform an LU factorization with complete pivoting.

Are there plans to include this in BLASFEO? Or is maybe someone already working on this? If there are no plans, but if it would be of interest, we are of course open to contribute.

**Comment by giaf** [comment]
There are no current plans to add LU factorization with complete pivoting, see also the response in https://github.com/giaf/blasfeo/discussions/146
